jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -89,24 +89,33 @@
       <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=functional --branch=mozilla-aurora --build-revision=$REVISION --build-locale=$LOCALE --installer-url=$INSTALLER_URL</commandLine>
+      <commandLine>python -u submission.py --test-type=functional --state=running --repository=mozilla-aurora --revision=$REVISION --locale=$LOCALE treeherder_venv</commandLine>
+      <executeFromWorkingDir>false</executeFromWorkingDir>
+    </hudson.plugins.xshell.XShellBuilder>
+    <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
+      <commandLine>python runtests.py --type=functional --installer-url=$INSTALLER_URL</commandLine>
       <executeFromWorkingDir>false</executeFromWorkingDir>
     </hudson.plugins.xshell.XShellBuilder>
   </builders>
   <publishers>
     <hudson.tasks.ArtifactArchiver>
-      <artifacts>*.xml,*.html,*.log</artifacts>
+      <artifacts>build/upload/*.*</artifacts>
```

Comment:
Everything to upload is located in the upload dir now. Both junit and html we no longer need given that we have treeherder.